### ISSUE TITLE
Bogus MD0036 error

### ISSUE THREAD

**Issue by XhmikosR** [DavidAnson/markdownlint#26]
Take this snippet for instance:

```markdown
* `io.js 1.6.2`, `io.js 1.6.3` x86 and x64 added. Use `Install-Product node 1` cmdlet to switch runtime to the latest io.js version.
* `Node.js 0.10.38`, `Node.js 0.12.2` x86 and x64 added. Use `Install-Product node 0` cmdlet to switch runtime to the latest Node.js version or `Install-Product node 0.10` to the latest 0.10.x build.
    * Node.js 0.10.38 x86 is default Node.js in `PATH`
* **Azure SDK 2.5.1**
* [Microsoft Visual Studio Installer Projects](https://visualstudiogallery.msdn.microsoft.com/9abe329c-9bba-44a1-be59-0fbf6151054d) extension installed (`.vdproj` support).
* [Bundler 1.9.2](https://rubygems.org/gems/bundler/versions/1.9.2) is pre-installed to all Rubies.
* **Python 2.7.9** replaces Python 2.7.8 in `C:\Python27` and `C:\Python27-x64`.
* **Python 3.4.3** replaces Python 3.4.1 in `C:\Python34` and `C:\Python34-x64`.
```

Another example:

```markdown
3. Run `install` scripts
4. Patch `AssemblyInfo` files
5. Modify `hosts` files
6. Start services
7. **Build**
    * Run `before_build` scripts
    * Run msbuild
    * Run `after_build` scripts
8. **Test**
```

That is an emphasis and not a header.

**Comment by XhmikosR** [comment]
Thanks, @DavidAnson!
